cogs/welcome/welcome.py

```diff
@@ -132,6 +199,138 @@ async def logServerLeave(self, leaveUser: discord.Member):
     async def welcome(self, ctx: Context):
         """Server welcome message settings."""
 
+    @checks.mod_or_permissions()
+    @welcome.command(name="channel")
+    async def welcomeChannelSet(self, ctx: Context, channel: discord.TextChannel):
+        """
+        Set the welcome channel
+
+        Parameters:
+        -----------
+        channel: The text channel to set welcome's to
+        """
+        if not isinstance(channel, discord.TextChannel):
+            return
+
```

Comment:
You can remove this. The bot actually helps do the type check to make sure we're getting a text channel before we enter this method.

Reply:
done